search-sentiments-map/public/index.html

```diff
@@ -49,29 +50,29 @@ <h2 id="title">Sentiment Map for Google Search</h2>
           <h5 id="topic-header">Worldwide sentiments</h5>
           <div id="map"></div>
           <div id="legend">
-            <div id="data-min">min</div>
-            <div class="color-key"></div>
             <div id="data-max">max</div>
+            <div class="color-key"></div>
+            <div id="data-min">min</div>
           </div>
```

Comment:
Why did you switch the order of these, logically doesn't it make sense to have data-min first?

Reply:
I swapped them because the way it renders is actually whatever is first on the left, and second on the right. So this sets it up so the min is on the left, the max is on the right.

Reply:
There were some issues with max and min being flipped throughout the code. Should be fixed now.